src/components/PeopleManagement/GroupDetailPage.jsx

```diff
@@ -0,0 +1,115 @@
+import { useIntl, FormattedMessage } from '@edx/frontend-platform/i18n';
+import {
+  Breadcrumb, Card, Hyperlink, Icon, IconButton, IconButtonWithTooltip, Toast, useToggle,
+} from '@openedx/paragon';
+import { Delete, Edit } from '@openedx/paragon/icons';
+
+import { useEnterpriseGroupUuid } from '../learner-credit-management/data';
+import { ROUTE_NAMES } from '../EnterpriseApp/data/constants';
+import DeleteGroupModal from './DeleteGroupModal';
+import EditGroupNameModal from './EditGroupNameModal';
+
+const GroupDetailPage = () => {
+  const intl = useIntl();
+  const { enterpriseSlug, groupUuid } = useParams();
+  const { data: enterpriseGroup } = useEnterpriseGroupUuid(groupUuid);
+  const [isDeleteModalOpen, openDeleteModal, closeDeleteModal] = useToggle(false);
+  const [isEditModalOpen, openEditModal, closeEditModal] = useToggle(false);
+  const [showToast, setShowToast] = useState(false);
+  const [toastMessage, setToastMessage] = useState('');
+
+  const tooltipContent = (
+    <FormattedMessage
+      id="admin.portal.budget.detail.tooltip.content"
+      defaultMessage="Delete group"
+      description="Tooltip to confirm meaning of trash icon."
+    />
+  );
+
+  const budgetNameAndEdit = (
+    <>
+      <span className="pr-1">{enterpriseGroup?.name}</span>
+      <IconButton
+        key="editGroupTooltip"
+        src={Edit}
+        iconAs={Icon}
+        alt="Edit group"
+        className="mr-2"
+        onClick={openEditModal}
+        size="sm"
+        data-testid="edit-modal-icon"
+      />
+    </>
+  );
+  return (
+    <div className="pt-4 pl-4">
+      <Toast
+        onClose={() => setShowToast(false)}
+        show={showToast}
+      >
+        {toastMessage}
+      </Toast>
+      <DeleteGroupModal
+        group={enterpriseGroup}
+        isOpen={isDeleteModalOpen}
+        close={closeDeleteModal}
+        setShowToast={setShowToast}
+        setToastMessage={setToastMessage}
+      />
+      <EditGroupNameModal
+        group={enterpriseGroup}
+        isOpen={isEditModalOpen}
+        close={closeEditModal}
+        setShowToast={setShowToast}
+        setToastMessage={setToastMessage}
+      />
+      <Breadcrumb
+        aria-label="people management breadcrumb navigation"
+        links={[
+          {
+            label: intl.formatMessage({
+              id: 'admin.portal.people.management.page.breadcrumb',
+              defaultMessage: 'People Management',
+              description: 'Breadcrumb label for the people management page',
+            }),
+            href: `/${enterpriseSlug}/admin/${ROUTE_NAMES.peopleManagement}`,
+          },
+        ]}
+        activeLabel={enterpriseGroup?.name}
+      />
+
+      <Card orientation="horizontal">
+        <Card.Body>
+          <Card.Header
+            title={budgetNameAndEdit}
+            subtitle={`${enterpriseGroup?.acceptedMembersCount} accepted members`}
+          />
+          <Card.Section className="pt-1 x-small">Created on</Card.Section>
```

Comment:
Need to add in created to the serializer, will add this value in with my next ticket